### PR TITLE
Add metrices for alephium watcher

### PR DESCRIPTION
**Pull request by Lbqds**

### CODE REVIEW

File: node/pkg/alephium/watcher.go [hunk 1]
```diff
@@ -211,6 +248,7 @@ func (w *Watcher) handleConfirmedEvents(logger *zap.Logger, confirmed []*Confirm
 		return nil
 	}
 
+	alphMessagesConfirmed.Add(float64(len(confirmed)))
```
Comment (h0ngcha0) [inline node/pkg/alephium/watcher.go:251]: Question: do we wanna do the same in `handleEvents` ?
Or maybe a general question, what is the relationship between `handleObsvRequest` and `fetchEvents/handleEvents` events?

**Review by h0ngcha0 — COMMENTED** [review]
LGTM :+1: 
Just a question for my understanding.

File: node/pkg/alephium/watcher.go (hunk 1 above)
Comment (Lbqds) [reply node/pkg/alephium/watcher.go:251]: There are 4 goroutines in the watcher:

1. `fetchHeight`: used to get the current block height, we can check if the event is confirmed by block height
2. `fetchEvents`: used to fetch events from the full node, and send the unconfirmed events to the `handleEvents` goroutine
3. `handleEvents`: wait for event confirmation, and remove events from the orphan blocks, then confirmed events will be handled in `handleConfirmedEvents` function, so we only need to update the `alphMessagesConfirmed` metrics once
4. `handleObsvRequest`: fetch the event from the full node and re-execute the consensus. It is only used when the event on the chain is not monitored due to unexpected circumstances.

Comment (h0ngcha0) [reply node/pkg/alephium/watcher.go:251]: Thanks for the explaination! <3

**Review by h0ngcha0 — APPROVED** [review]
LGTM :+1:

**Comment by h0ngcha0** [comment]
ALPH metrics seem to be working nicely
https://grafana.softfork.se/d/-GcztLMnz/wormhole-v2?orgId=1

**Review by polarker — APPROVED** [review]
LGTM!

**Pull request merged**